subworkflows/local/call_repeat_expansions.nf

```diff
@@ -0,0 +1,24 @@
+//
+// Run ExpansionHunter
+//
+
+params.expansionhunter_options = [:]
+
+include { EXPANSIONHUNTER } from '../../modules/nf-core/modules/expansionhunter/main'  addParams( options: params.expansionhunter_options )
+
+workflow CALL_REPEAT_EXPANSIONS {
```

Comment:
@ljmesi Can we refactor this to make it more general? @jemten What do you think?
```suggestion
workflow CALL_SHORT_TANDEM_REPEATS {
```

Reply:
To me both are rather general, but go for it if you like 😄

Reply:
Okay then we scrap my suggestion if it's already general enough 👍 😄